index.ftd

```diff
@@ -456,28 +459,46 @@ ftd.image-src src:
 optional ftd.align alignment: center
 optional ftd.resizing width:
 optional ftd.resizing height:
+optional ftd.image-src src2:
```

Comment:
```suggestion
optional ftd.image-src src2:
```
@saurabh-lohiya give meaningful property name, what do you mean by src2? is this the way world will use it?